staker/stateless_block_validator.go

```diff
@@ -428,8 +429,13 @@ func (v *StatelessBlockValidator) Start(ctx_in context.Context) error {
 			return fmt.Errorf("starting execution spawner: %w", err)
 		}
 	}
-	for _, spawner := range v.execSpawners {
+	for i, spawner := range v.execSpawners {
 		if err := spawner.Start(ctx_in); err != nil {
+			if u, parseErr := url.Parse(v.config.ValidationServerConfigs[i].URL); parseErr == nil {
+				if u.Scheme == "http" || u.Scheme == "https" {
+					return fmt.Errorf("validation server's url scheme: %s is unsupported, it should either be ws or wss, err: %w", u.Scheme, err)
```

Comment:
I'm thinking error message should include full URL instead of just the scheme.  Also, `should be either http or https` instead of `ws or wss`